device/thunder/src/lib.rs

```diff
@@ -34,3 +34,8 @@ pub mod processors {
     pub mod thunder_wifi;
     pub mod thunder_window_manager;
 }
+
+pub mod tests {
```

Comment:
Lets modularize this a bit further 
create a folder for contracts
-- contracts
-- device_info_ptest.rs

Reply:
device_info means only test case related to device.info method or all the device thunder tests??

Reply:
done